.gitattributes

```diff
@@ -18,6 +18,8 @@ CONTRIBUTING.md          ocaml-typo=missing-header
 INSTALL                  ocaml-typo=missing-header
 LICENSE                  ocaml-typo=non-printing,missing-header
 
+Makefile.shared          ocaml-typo=long-line
```

Comment:
OK - out of curiosity only, what are they? I found years ago working with `MicrosoftLongNamedAPI(VERY_LONG_NAMED_CONSTANT,)` resulted in 80 characters vastly reducing readability in too many places and so switched to 120-150 (especially as screens got wider, rather than longer)